hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -101,7 +114,12 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
         buckets: clusterState.buckets,
         keys: clusterState.keys,
         missingContainersCount,
-        lastUpdated: Number(moment())
+        lastUpdated: Number(moment()),
+        lastUpdatedOM: taskStatus[0] ? taskStatus[0] && taskStatus[0].lastUpdatedTimestamp : 0,
+        lastUpdateOMSync:taskStatus[1] ? taskStatus[1] && taskStatus[1].lastUpdatedTimestamp : 0,
+        lastUpdatedOMText:taskStatus[0] ? taskStatus[0] && taskStatus[0].taskName.toLowerCase() === 'OmDeltaRequest'.toLowerCase() ? 'Last Delta Update': 'Last Full Update': '',
```

Comment:
I believe we could simply use `Last Delta Update` once we separate the two variables?

Reply:
Done Changes